distr_tracing/distr_tracing_rn/distr-tracing-rn-2-8.adoc

```diff
@@ -0,0 +1,58 @@
+:_content-type: ASSEMBLY
+[id="distributed-tracing-rn-2-8"]
+= Release notes for {DTProductName} 2.8
+include::_attributes/common-attributes.adoc[]
+:context: distributed-tracing-rn-2-8
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+== Component versions in the {DTProductName} 2.8
+
+[options="header"]
+|===
+|Operator |Component |Version
+|{JaegerName}
+|Jaeger
+|1.42
+
+|{OTELName}
+|OpenTelemetry
+|0.74.0
+
+|{TempoName}
+|Tempo
+|0.1.0
+|===
+
+== Technology Preview features
+
+This release introduces support for the {TempoName} as a link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview] feature for {DTProductName}.
+The feature uses version 0.1.0 of the {TempoName} and version 2.0.1 of the upstream {TempoShortName} components.
+
+You can use the {TempoShortName} to replace Jaeger so that you can use S3-compatible storage instead of ElasticSearch.
+Most users who use the {TempoShortName} instead of Jaeger will not notice any difference in functionality because the {TempoShortName} supports the same ingestion and query protocols as Jaeger and uses the same user interface.
+
+If you enable this Technology Preview feature, note the following limitations of the current implementation:
+
+* The {TempoShortName} currently does not support disconnected installations. (link:https://issues.redhat.com/browse/TRACING-3145[TRACING-3145])
+
+* When you use the Jaeger user interface (UI) with the {TempoShortName}, the Jaeger UI lists only services that have sent traces within the last 15 minutes. For services that have not sent traces within the last 15 minutes, those traces are still stored even though they are not visible in the Jaeger UI. (link:https://issues.redhat.com/browse/TRACING-3139[TRACING-3139])
+
+Expanded support for the {TempoOperator} is planned for future releases of the {DTProductName}.
+Possible additional features might include support for TLS authentication, multitenancy, and multiple clusters.
+For more information about the {TempoOperator}, see the link:https://tempo-operator.netlify.app[Tempo community documentation].
+
+:FeatureName: The {TempoName}
+include::snippets/technology-preview.adoc[leveloffset=+1]
```

Comment:
That standard practice is to mention the tech preview snippet at the start and then mention details. Consider moving this at the beginning of the section and not at the end.

Reply:
Thank you. Done as suggested.